frontend/src/components/modules/routeDetail/RouteDetail.tsx

```diff
@@ -74,16 +75,7 @@ function RouteDetail({scheduleDefinitionId}: routeDetailProps) {
                     </Box>
                 </Box>
             </Box>) :
-            (<Box padding={1} width={'100%'} display={'flex'} flexDirection={'column'}>
-                <Box padding={1} marginBottom={2} bgcolor={'var(--secondary-light)'}
-                    borderRadius={'var(--small_corner)'}
-                    display={'flex'}>
-                    <Box>
-                        <Typography variant={'h4'}>Geen route geselecteerd</Typography>
-                        <Typography variant={'subtitle1'}>Selecteer een route om details weer te geven</Typography>
-                    </Box>
-                </Box>
-            </Box>));
+            (<NoneSelected ElementName={'route'}/>));
```

Comment:
For the buildings, live routes, and users, the check is done in the container instead of the detail component. Maybe it would be cleaner to do this for routes as well by moving this check to the RoutsPage container and removing it from the detail component?